nipype/pipeline/plugins/pbs.py

```diff
@@ -46,13 +45,12 @@ def __init__(self, **kwargs):
         super(PBSPlugin, self).__init__(template, **kwargs)
 
     def _is_pending(self, taskid):
-        #  subprocess.Popen requires taskid to be a string
-        proc = subprocess.Popen(["qstat", str(taskid)],
-                                stdout=subprocess.PIPE,
-                                stderr=subprocess.PIPE)
-        _, e = proc.communicate()
+        result = CommandLine('qstat {}'.format(taskid),
+                             environ=dict(os.environ),
+                             terminal_output='allatonce',
```

Comment:
I'm not sure `'allatonce'` generates both `stdout` and `stderr` in the runtime object. Have you checked?